content/pages/vocational-rehab-and-employment/apply-vre.md

```diff
@@ -1,26 +1,11 @@
 ---
 layout: page-breadcrumbs.html
 title: Apply for VR&amp;E Benefits
-template: 4-action-page
+template: 6-info-page
 ---
 
-<div class="main" role="main" markdown="0">
-
-
-<div class="section one" markdown="0">
-<div class="primary" markdown="0">
-<div class="row" markdown="0">
-<div class="small-12 medium-8 columns">
-
-
-
-<div markdown="1">
```

Comment:
Removing unnecessary divs surrounding content. This can get pretty in the weeds, but a general guideline is you should only need to put div tags before and after a chunk of content when you're doing some type of special styling like a blue callout box or subway map.